cpp/examples/abm_minimal.cpp

```diff
@@ -165,10 +147,20 @@ int main()
     mio::abm::close_social_events(t_lockdown, 0.9, world.parameters);
 
     auto t0   = mio::abm::TimePoint(0);
-    auto tmax = mio::abm::TimePoint(0) + mio::abm::days(30);
+    auto tmax = mio::abm::TimePoint(0) + mio::abm::days(10);
     auto sim  = mio::abm::Simulation(t0, std::move(world));
 
     sim.advance(tmax);
 
-    write_results_to_file(sim);
+    std::ofstream outfile("abm_minimal.txt");
+
+    // The results are saved in a table with 9 rows.
+    // The first row is Time. The other rows correspond to the number of people with a certain infection state at this Time:
```

Comment:
```suggestion
    // The results are saved in a table with 9 columns.
    // The first column is Time. The other columns correspond to the number of people with a certain infection state at this Time:
```
I think this got transposed somewhere ;)